articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,160 @@
+- [What is inter-VLAN routing?](#what-is-inter-vlan-routing)
+- [Methods of inter-VLAN routing](#inter-vlan-routing-methods)
+- [Inter-VLAN network configurations](#network-configurations-for-inter-vlan-communication-using-router-on-stick-method)
+- [Conclusion](#conclusion)
+
+### Prerequisite
+As a prerequisite, the reader must have a good understanding of basic networking concepts, and how to work with Cisco packet tracer.
+
+To download Cisco packet tracer visit [this link](https://www.computernetworkingnotes.com/ccna-study-guide/download-packet-tracer-for-windows-and-linux.html) to install the latest version installed on your computer.
+
+### What is Inter-VLAN routing?
+Virtual Local Area Networks (VLANs) are network segments on a switched LAN that groups different hosts together logically, regardless of their physical locations on the network.
+
+VLANs make it easier to segment a workstation into different isolated LANs. They improve the performance of the network and are flexible, since they are logically connected.
+
+VLANs act as separate subnets on the network. To move packets from one VLAN to another and enable communications between hosts, we have to configure the network to enable inter-VLAN routing.
+
+### Inter-VLAN routing methods
+#### Legacy inter-VLAN routing
+In this method, multiple router interfaces are used, each connecting to a switch port in different VLANs. These interfaces are served as default gateways. Sometimes, it may lead to network issues, since expanding the network requires additional cabling.
+
+#### Router-on-a-stick
+In this method, unlike the legacy routing, one physical interface port is used for routing the traffic between the network segments. The network administrator need not create separate VLAN interfaces like `fa0/1` to `fa0/10`. Instead, all the interfaces from 1 to 10 are created with a single interface.
+
+#### Layer 3 switch using Switched Virtual Interface (SVI)
+Currently, this method of inter-VLAN routing that uses layer 3/multilayer switch and Switched Virtual Interfaces (SVI) are being used more commonly. SVIs are created for VLANs that exist on the switch and they perform the same function for the VLANs as that of a router.
+
+Layer 3 switches are expensive, hence not affordable.
+
+### Network configurations for Inter-VLAN communication using Router-On-Stick method
+In this article, we will learn how to configure inter-VLAN routing using the router-on-a-stick method.
+
+Consider a LAN with 4 PCs, 1 switch, and a router connected as shown in the image:
+
+![Local Area Network](/engineering-education/inter-vlan-routing/network.jpg)
+
+We want to configure two VLANs 10 and 20. PC0 and PC1 on VLAN10 while PC2 and PC3 are on VLAN20.
+* IP Address of PC0   -192.168.1.10
+* IP Address of PC1   -192.168.1.20
+* IP Address of PC2   -192.168.2.10
+* IP Address of PC3   -192.168.2.20
+* The default gateway for VLAN10- 192.168.1.1
+* The default gateway for VLAN10- 192.168.2.1
```

Comment:
For bullet points, do not use `*`, instead use `-`. Something like this

- IP address of PC0

Reply:
Make these changes wherever necessary